yarn.lock

```diff
@@ -3814,6 +3814,13 @@ axe-core@^4.0.2:
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-4.1.1.tgz#70a7855888e287f7add66002211a423937063eaf"
   integrity sha512-5Kgy8Cz6LPC9DJcNb3yjAXTu3XihQgEdnIg50c//zOC/MyLP0Clg+Y8Sh9ZjjnvBrDZU4DgXS9C3T9r4/scGZQ==
 
+axios@^0.21.1:
```

Comment:
@cotts axios is still in `yarn.lock`. Can you remove it by `yarn remove axios` then commit changes in `yarn.lock`?

Reply:
removed